### DISCUSSION TITLE
Color Correction of Input Images

### DISCUSSION THREAD

**Discussion by simon-cross** [cdcseacave/openMVS#767] [Q&A]
I am looking for a color correction method for multi-view images and I am wondering whether openMVS implements one of these methods. If yes, I could I run it and get the color corrected input images?

**Comment by cdcseacave** [comment]
Yes, there are 2 color correction methods implemented in mesh texturing module, one that adjust global exposure to bring images close to eachother and a second that blends the seams between images once placed next to eachother. However there is none applied to the entire images, though the first one could be used. There are other papers proposing variations on this idea.
